MOLT Fetch GA documentation (#18541)
cockroachdb/docs: src/current/v24.1/molt-verify.md

Complete the following items before using MOLT Verify:

> - Make sure the SQL user running MOLT Verify has [`SELECT` privileges]({% link {{ page.version.version }}/grant.md %}#supported-privileges) on the source and target CockroachDB tables.

Reviewer: ```suggestion
- The SQL user running MOLT Verify must have the [`SELECT` privilege]({% link {{ page.version.version }}/grant.md %}#supported-privileges) on both the source and target CockroachDB tables.
```
